modules/gdnative/pluginscript/pluginscript_script.cpp

```diff
@@ -36,15 +36,13 @@
 
 #ifdef DEBUG_ENABLED
 #define __ASSERT_SCRIPT_REASON "Cannot retrieve pluginscript class for this script, is you code correct ?"
```

Comment:
Capitalize "PluginScript", fix `s/you/your/` and remove the French space before "?" :)